webapp/client/src/stories/FormNavButtons.stories.js

```diff
@@ -21,6 +21,11 @@ WithOverviewLink.args = {
 export const WithExplanatoryText = Template.bind({});
 WithExplanatoryText.args = {
   ...Default.args,
-  explanatoryButtonText:
-    'Sie haben Ihren Freischaltcode bereits erhalten? <br><a href="#">Dann können Sie sich anmelden</a>',
+  explanatoryButtonText: (
+    <>
+      Sie haben Ihren Freischaltcode bereits erhalten?
+      <br />
```

Comment:
I was curious why you had to change `<br>` to `<br />` and that was the result of my five minute research into the topic. 

- In HTML5 self-closing tags which can not have a child can use an optional slash at the end for compatibility with XHTML. However, the browser will ignore it. ([s. here](https://www.tutorialstonight.com/self-closing-tags-in-html.php))
- In Reacts/JSX you need to include the slash ([s. here](https://www.codecademy.com/courses/react-101/lessons/react-jsx-advanced/exercises/self-closing-tags)). Unfortunately, I could not find the reason besides that JSX is closer to XML than HTML. I am not going to invest more time into that but if you know it, let me know :) 


**TLDR: If you use self-closing tags like `<br>` in JSX, include the slash.**

Reply:
Good to know! This was an automatic change made by prettier.